src/hooks/useUnmountingAnimation.ts

```diff
@@ -1,9 +1,29 @@
 import { RefObject, useEffect } from 'react'
 
+/**
+ * Checks whether a given node is currently animating.
+ *
+ * @param {Animatable | Document} node - The node to check for ongoing animations.
+ * @returns {boolean} True if the node is animating; false otherwise.
+ */
 function isAnimating(node?: Animatable | Document) {
```

Comment:
Use TypeScript typings to self-document types - avoid re-documenting in JSDoc so there is less chance of docs going stale.

```suggestion
 * @param node - The node to check for ongoing animations.
 * @returns - true if the node is animating; false otherwise.
 */
function isAnimating(node?: Animatable | Document): boolean {
```